tfrog-motordriver/communication.c

```diff
@@ -1287,6 +1287,13 @@ int extended_command_analyze(char* data)
     send(data);
     send("\n00P\n\n");
   }
+  else if (strstr(data, "$SETBAUDRATE") == data)
```

Comment:
`$SET485BAUDRATE` would be better as it also has debug serial and USB tty device.